apps/backport/datasync/data_metadata.py

```diff
@@ -124,11 +121,11 @@ def _load_variable(session: Session, variable_id: int) -> Dict[str, Any]:
     """
 
     # Using the session to execute raw SQL and fetching one row as a result
-    result = session.execute(sql, {"variable_id": variable_id}).fetchone()  # type: ignore
+    result = session.execute(text(sql), {"variable_id": variable_id}).fetchone()
```

Comment:
i see how we can create a wrapper that applies `text(...)` to all entries. Maybe a simple extension of `Session`. Not urgent at all, can do in the future

Reply:
Yeah, I was thinking about that too and was actually surprised that SQLAlchemy doesn't already have such a helper function. (Feel free to add a helper function if you find yourself repeating it over and over again)